### DISCUSSION TITLE
使用阿里云引擎会不会调用专业版

### DISCUSSION THREAD

**Discussion by leesocool** [YiiGuxing/TranslationPlugin#2275] [Q&A] [answered]
我的阿里云有通用版和专业版两个引擎，两个都有免费额度，麻烦问一下，通用版额度用完了会使用专业版的额度吗
![image](https://user-images.githubusercontent.com/62526571/196876610-eac8d9f0-bc52-4738-af1b-704853297721.png)

**Comment by YiiGuxing** [comment, marked as answer]
目前不会，目前是通用版本的。
